examples/pytorch/graphsage/main.py

```diff
@@ -170,7 +141,7 @@ def train_func(config: Dict):
 
 def run_ray(init_dataset_fn, **kwargs):
     """Run ray trainer."""
-    ray.init(num_cpus=3)
+    ray.init(num_cpus=4)
```

Comment:
We probably should autodetect cpus..

Reply:
This is only necessary because CI pipeline does not have enough cpus.